go.mod

```diff
@@ -97,7 +97,7 @@ require (
 	gonum.org/v1/gonum v0.6.2
 	gonum.org/v1/netlib v0.0.0-20190331212654-76723241ea4e // indirect
 	google.golang.org/api v0.46.0
-	google.golang.org/genproto v0.0.0-20220107163113-42d7afdf6368
+	google.golang.org/genproto v0.0.0-20220310185008-1973136f34c6
```

Comment:
run `hack/lint-dependencies.sh` and follow the instructions to make sure transitive deps are updated as well